dashboard/src/actions/datasetListActions.js

```diff
@@ -1,13 +1,15 @@
 import * as TYPES from "./types";
 
 import API from "../utils/axiosInstance";
+import { uriTemplate } from "utils/helper";
```

Comment:
Nit: `overviewActions.js` and `tableOfContentActions.js` import from `../utils/helper` (although `findNoOfDays` is imported from `utils/dateFunctions` in `overviewActions.js`). Assuming that they are equivalent, maybe we should go with a single idiom in all cases.

Reply:
It's a good point, and I'm not quite sure how this ended up that way because I'm usually not inclined to depend on the default root. (And it confuses `vscode`, too.)